integration/src/bitcoin/bitcoin.ts

```diff
@@ -61,6 +61,59 @@ export function bitcoinTests (get: () => { wallet: HDWallet, info: HDWalletInfo
       expect(await info.btcSupportsCoin('Testnet')).toBeTruthy()
     }, TIMEOUT)
 
+    test.only('getPublicKeys', async () => {
+      if (!wallet) return
+      expect(await wallet.getPublicKeys([{
+        coin: 'Bitcoin',
+        addressNList: bip32ToAddressNList(`m/44'/0'/0'`),
+        curve: 'secp256k1'
+      }, {
+        coin: 'Bitcoin',
+        addressNList: bip32ToAddressNList(`m/49'/0'/0'`),
+        curve: 'secp256k1',
+        scriptType: BTCInputScriptType.SpendAddress
+      }, {
+        coin: 'Bitcoin',
+        addressNList: bip32ToAddressNList(`m/49'/0'/0'`),
+        curve: 'secp256k1',
+        scriptType: BTCInputScriptType.SpendP2SHWitness
+      }, {
+        coin: 'Bitcoin',
+        addressNList: bip32ToAddressNList(`m/49'/0'/0'`),
+        curve: 'secp256k1',
+        scriptType: BTCInputScriptType.SpendAddress
+      }, {
+        coin: 'Bitcoin',
+        addressNList: bip32ToAddressNList(`m/84'/0'/0'`),
+        curve: 'secp256k1',
+        scriptType: BTCInputScriptType.SpendWitness
+      }, {
+        coin: 'Bitcoin',
+        addressNList: bip32ToAddressNList(`m/0'/0'/0'`),
+        curve: 'secp256k1',
+        scriptType: BTCInputScriptType.SpendAddress
+      }, {
+        coin: 'Litecoin',
+        addressNList: bip32ToAddressNList(`m/0'/0'/0'`),
```

Comment:
FIXME: would also be nice to test paths like `m/`, `m/42'`, etc, but the current implementation in `hdwallet-ledger` assumes addressNList has length == 3.

Reply:
Nice to have...

Reply:
we fixed that bug though